crates/node-bindings/src/helpers/js_callable/serde.rs

```diff
@@ -0,0 +1,34 @@
+use napi::bindgen_prelude::Array;
+use napi::bindgen_prelude::FromNapiValue;
+use napi::Env;
+use napi::JsUnknown;
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+
+pub fn map_params_serde<Params: Serialize + Send + Sync + 'static>(
+  params: Params,
+) -> Box<dyn FnOnce(&Env) -> napi::Result<Vec<JsUnknown>> + Send + 'static> {
```

Comment:
Why do we return `Vec<...>` instead of `JsUnknown`?

Reply:
To describe multiple arguments for the target JavaScript function. This is how the Napi API works so it just maps directly to it